packages/design-tokens/src/tailwind/tailwind-presets.ts

```diff
@@ -0,0 +1,126 @@
+// ts-check
+// import {Config} from "tailwindcss"
```

Comment:
Currently using this to incrementally check the updated presets, before ready for production I think it would be best to create our own type or interface. If we were to use the native config object we would also have to make tailwind a dependency of our design-tokens, which I don't know if we'd want that